lint/README.md

```diff
@@ -4,10 +4,12 @@
 
 These will be part of reviewing PRs to this repo:
 
-1. Avoid adding dependencies to rules_lint, they belong in the example instead. For example in adding
-   eslint or flake8, it's up to the user to provide us the binary to run.
-   This ensures that the user can select the versions of their tools and the toolchains used to run them
-   rather than us baking these into rules_lint.
+1. Take care with dependencies. Avoid adding to `MODULE.bazel` if possible.
+   In cases where a tool is a statically-linked binary, it can be added to the `multitool.lock.json` file
+   to conveniently provide it to users.
+   In other cases where a language ecosystem's package manager is involved,
+   the tool should be setup "in userland", which means adding it to the `example` folder.
+   Note that this distinction also determines whether users control the version of the tool.
```

Comment:
For my education, do all formatters provide an escape hatch mechanism so that the user can provide their own version? Maybe add a bullet point saying yes or no to that since other users might wonder

Reply:
Yes, in the sense that users can fetch whatever tool in their own repo, then provide that label in their `format_multirun` call. I'll add a docstring about it though.